plugins/catalog/src/plugin.ts

```diff
@@ -186,3 +186,15 @@ export const EntitySystemDiagramCard = catalogPlugin.provide(
     },
   }),
 );
+
+export const EntityGenericRelationshipCard = catalogPlugin.provide(
```

Comment:
Just plain `RelatedEntitiesCard` is fine with me.

Seems you forgot to export it all the way though - I expected to see a change in `index.ts` and a corresponding generated change in `api-report.md`.

Reply:
Updated it, don’t know why the goalie workflow failed though.